problems/strings.js

```diff
@@ -87,14 +136,23 @@ function removeLastWord() {}
  * @returns {string}
  */
 
-function everyOtherLetter() {}
+function everyOtherLetter(string) {
+  let newString = "";
+  for (let i = 0; i < string.length; i += 2) {
+    newString += string[i];
+  }
+  return newString;
+}
 
+// console.log(everyOtherLetter("beautiful"))
 /**
  * Takes in a string and returns a new string that has all the vowels removed.
  * @param {string}
  * @returns {string}
  */
-function disemvowel() {}
+function disemvowel(string) {
+  return string.replace(/[aeiou]/gi, "");
```

Comment:
Slick way of writing this. Make sure you understand this so that you could do it in an interview.